extensions/ui/org.eclipse.smarthome.ui.classic/src/main/java/org/eclipse/smarthome/ui/classic/internal/render/SelectionRenderer.java

```diff
@@ -44,6 +45,8 @@
 
     private final Logger logger = LoggerFactory.getLogger(SelectionRenderer.class);
 
+    private String mappedValue;
```

Comment:
Do you really want that as an instance field? Isn't that an issue if there are concurrent requests that render different widgets with mappings at the very same moment?

Reply:
That was an easy way as I have now a method that should return 2 data ... but not clean.
I will try something else.